### PR TITLE
Add Kyverno content and mappings

### PR DESCRIPTION
**Pull request by MaxRink**
## Summary
- add a Mondoo Kyverno policy for PolicyException hygiene and Kyverno failing results
- honor Kyverno scanner strictness settings for expired and unmapped PolicyExceptions
- split Kyverno PolicyReport failures into mapped and unmapped checks so unmapped results remain visible by default but can be disabled with `kyverno-report-unmapped-policy-results`
- replace the small mapping seed with a pinned official `kyverno/policies` catalog snapshot
- catalog all 337 unique upstream Kyverno policy names from 645 policy resources at ref `76be98a25d49ae01278a94ecde8f50f9e08577ef`
- add 94 rule-level mapping entries where Mondoo Kubernetes checks overlap, with mapped, partial, and unmapped policy coverage tracked in tests
- include exact checks and official mappings for PodSecurity, RBAC, Secret, Service, Ingress, NetworkPolicy, workload, and resource-posture controls
- include `scripts/kyverno-mapgen` to regenerate the pinned catalog from a local `kyverno/policies` checkout while preserving reviewed mappings

## Review fixes
- align the Job/CronJob check with the official Kyverno first-owner-reference semantics
- downgrade RBAC escalation-verbs and Secret-read-verbs mappings to partial because the Mondoo checks are intentionally broader than the official Kyverno policies
- mark `check-hpa-exists` partial and exclude DaemonSets from native HPA posture because native HPAs do not scale DaemonSets
- clarify the image-pull-secret check title so it matches the actual `ghcr.io` / `quay.io` allowlist semantics
- remove stale personal-fork MQL checksums from `go.sum`
- explicitly load required core/network/os/k8s schema fixtures in content tests and add regression assertions for mapping precision and query edge cases

## Validation
- `git diff --check`
- `go test ./content -run 'TestKyvernoMappedKubernetesChecksCoverEdgeCases|TestKyvernoOfficialPolicyMappingsReferenceExistingMondooChecks|TestKyvernoOfficialPolicyMappingSummary|TestKyvernoMappedKubernetesBundlesCompile|TestMondooKubernetesKyvernoPolicyBundleLoads'`
- `go test ./content`

## Dependencies
Draft depends on the MQL provider PR that introduces `k8s.kyverno` resources, mappings, and Kyverno behavior settings. A normal cnspec release/provider bundle will be needed before users can run this content without an explicit local provider cache.

### CODE REVIEW

File: content/kyverno_mappings_test.go [hunk 1]
```diff
@@ -0,0 +1,741 @@
...
+	_, err := os.Stat(path)
+	require.NoErrorf(t, err, "in-repo schema fixture must point to a readable network.lr schema file")
+	return path
+}
+
+func kyvernoOSSchemaPath(t *testing.T) string {
+	t.Helper()
+
+	path := "./testdata/schema/providers/os/resources/os.lr"
+	_, err := os.Stat(path)
```
[the remark this answers is not in the document]
Comment (MaxRink) [reply content/kyverno_mappings_test.go]: Addressed in e475035b: collapsed the four `kyverno*SchemaPath` helpers into a single parameterized `kyvernoSchemaPath(t, provider)` that builds the path from the provider name and uses `%s.lr` in the error message, exactly as suggested. All four call sites now pass the provider string.

**Comment by MaxRink** [comment]
Rebased onto `origin/main` (now at v13.30.0) and pushed as `e475035b`.

**Conflict resolution:** The only conflict was in `go.mod` — a dependency-version clash. I took upstream's newer versions (mql v13.30.0, ranger-rpc v0.8.1, go-cty v1.19.0, updated golang.org/x/* and google.golang.org/genproto). The PR had added `go.yaml.in/yaml/v3 v3.0.4` as a direct require while upstream carried it as indirect; `go mod tidy` reconciled this by promoting it to a direct dependency at upstream's v3.0.4 (the generator and test import it directly). No PR features were dropped.

**Review comment addressed:** The one unresolved thread — collapsing the four near-identical `kyverno*SchemaPath` helpers — is fixed. They are now a single `kyvernoSchemaPath(t, provider string)` that derives the path and error message from the provider name. The other five threads were already marked resolved.

**Build/test:** `go build ./...` passes. `go test ./scripts/kyverno-mapgen/` and the fixture-based content tests (`TestKyvernoOfficialPolicyMappingsReferenceExistingMondooChecks`, `TestMondooKubernetesKyverno*`, `TestKyvernoMappedKubernetesChecksCoverEdgeCases`) all pass locally.

**Note:** `TestKyvernoMappedKubernetesBundlesCompile` fails in my local environment because it uses `providers.DefaultRuntime()` (the installed k8s provider), and my locally-installed k8s provider predates the companion `k8s.kyverno` resource. This is unchanged from the original commit and unrelated to the rebase; CI installs the provider build that exposes `k8s.kyverno`.
